src/toil/batchSystems/abstractGridEngineBatchSystem.py

```diff
@@ -267,7 +268,7 @@ def _runStep(self):
             if self.checkOnJobs():
                 activity = True
             if not activity:
-                logger.debug('No activity, sleeping for %is', self.boss.sleepSeconds())
+                logger.log(TRACE, 'No activity, sleeping for %is', self.boss.sleepSeconds())
```

Comment:
Should this go back to debug?

Reply:
No, I lowered this to TRACE since I didn't think it made sense to log every second at debug level.